src/database/user_operations.py

```
@@ -190,4 +190,48 @@ async def admin_delete_account(pool, email):
+                (amount, mu)
+            )
+            affected = cur.rowcount
+            
+            if affected:
+                # Получаем новый баланс
+                await cur.execute("SELECT coins FROM account WHERE email=%s", (mu,))
+                new_balance = (await cur.fetchone())[0]
+                logger.info(f"Добавлено {amount} монет на аккаунт {mu}. Новый баланс: {new_balance}")
+                return new_balance
+            else:
+                logger.warning(f"Не удалось добавить монеты на аккаунт {mu}")
```

Ruff (0.12.2)

String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)